app/controllers/support_interface/errors_controller.rb

```diff
@@ -0,0 +1,10 @@
+module SupportInterface
```

Comment:
Interesting question. Neither of these have their own header in the code, so they get the standard 404 copy with the generic 'Apply...' header, which takes you back to candidate.

I could add some lines in the header partial so that they get their own header (what would `service_name` be in these cases?) linking somewhere sensible (`VendorAPI` 404s could link to the api-docs, but what should `RefereeInterface` link to?)

Reply:
I guess `RefereeInterface` could link to candidate and that wouldn't be the end of the world? And `service_name` should probably support `VendorAPI`?

Reply:
I think for `VendorAPI` we should do something like https://github.com/DFE-Digital/apply-for-teacher-training/pull/2706 but maybe support both html and json responses, just in case users type API urls in their browser. Perhaps best handled as a separate PR, after the new `VendorAPI` 500 handling is in.

`RefereeInterface` I'll leave as is, it already links to the candidate interface. But I'm waiting for some content tweaks before this is mergeable.